packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "../interface/FundingRateStoreInterface.sol";
+import "../../../common/implementation/Testable.sol";
+
+
+contract FundingRateStore is FundingRateStoreInterface, Testable {
+    struct Proposal {
+        FixedPoint.Unsigned rate;
+        uint256 time;
+        address proposer;
+    }
+
+    struct FundingRateRecord {
+        FixedPoint.Unsigned rate;
+        Proposal proposal;
+    }
+
+    enum ProposalState { None, Pending, Expired }
+
+    mapping(bytes32 => FundingRateRecord) public fundingRateRecords;
+
+    uint256 proposalLiveness;
+
+    constructor(address _timerAddress) public Testable(_timerAddress) {}
+
+    function getLatestFundingRateForIdentifier(bytes32 identifier)
```

Comment:
This interface method was renamed to `getFundingRateForIdentifier`

Reply:
Gotcha. Will update.